schemas/com.snowplowanalytics.badrows/enrichment_failures/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a bad row resulting from enrichment failures",
+  "self": {
+    "vendor": "com.snowplowanalytics.badrows",
+    "name": "enrichment_failures",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "failure": {
+      "type": "object",
+      "description": "Information regarding the enrichment failures",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "description": "Timestamp at which the failure occurred",
+          "format": "date-time"
+        },
+        "messages": {
+          "type": "array",
+          "description": "List of failure messages associated with the enrichment failures",
+          "items": {
+            "type": "object",
+            "description": "Individual enrichment failure",
+            "anyOf": [
+              {
+                "properties": {
+                  "enrichment": {
+                    "type": "string",
+                    "description": "Enrichment which produced this failure",
+                    "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+                  },
+                  "message": {
+                    "type": "string",
+                    "description": "Failure message associated with this enrichment failure",
+                    "maxLength": 256
+                  }
+                },
+                "required": [ "enrichment", "message" ],
+                "additionalProperties": false
+              },
+              {
+                "properties": {
+                  "etc": {
+                    "type": "string"
+                  }
+                },
+                "required": [ "etc" ],
+                "additionalProperties": false
+              }
```

Comment:
we need to look into the enrichments in order to make this ADT either more extensive or restrictive.